.gitignore

```diff
@@ -18,6 +18,11 @@
 .env.test.local
 .env.production.local
 
+# ignore congig files for heroku to work properly
+
+.eslintrc.json
```

Comment:
You need to remove `.eslintrc.json` from this file, and also set up the recommended `.eslintrc.json`  check out the [microverse-linter-config](https://github.com/microverseinc/linters-config/tree/master/react-redux), according to the project requirement we need to set the linter's correctly with the standard Microverse config file without changing it.